scripts/generators/format_utils_generator.py

```diff
@@ -0,0 +1,695 @@
+#!/usr/bin/python3 -i
+#
+# Copyright 2023 The Khronos Group Inc.
+# Copyright 2023 Valve Corporation
+# Copyright 2023 LunarG, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from generators.vulkan_object import (Format)
+from generators.base_generator import BaseGenerator
+
+# Make C++ name friendly class name
+def getClassName(className: str) -> str:
+    return className.replace('-', '').replace(' ', '_').upper()
```

Comment:
Is this needed? We are generating a C header.

Reply:
Oh absolutely - its 'C++ & C' friendly.

Spec uses `16-bit 2-plane 420` whereas we need `16BIT_2PLANE_420`.